### ISSUE TITLE
List numbering

### ISSUE THREAD

**Issue by dtaylor7** [bpampuch/pdfmake#875]
When creating nested list the numbers jump a step. If you take a look at your [playground](http://pdfmake.org/playground.html) on the list example. you will see on the nested list example the numbers jump from 2 to 4.

e.g.
     1.  item 1
     2.  Lorem ipsum dolor sit amet, consectetur adipisicing elit. Malit profecta versatur nomine
ocurreret multavit
          nested list .....
     4.  item 3
          second line of item3

**Comment by stretch4x4** [comment]
I think this is actually correct as the third list item is a sub-list. However I think it would be good if there was an option to turn on and off the showing of sublist number/bullets as there are some cases where it is important to show them and some, where it is not.
EG:
`<ol><li><ol><li><ol><li>Item</li></ol></li></ol></li></ol>`
`1. 1. 1. Item`
Technically the following is invalid html but this is how chrome renders it (and the user is able to create it on occasion) so I need to make the pdf handle it the same way
`<ol><ol><ol><li>Item</li></ol></ol></ol>`
`      1. Item`
The current behaviour is helpful for me solving the latter case but I am not able to represent the first legal case currently

**Comment by Ruud-Zuiderlicht** [comment]
This is indeed other then I'd expect. But perhaps the playground needs updating because it is possible to get correct numbering using a stack (array) instead with the text and the nested list. The playground already includes that for the "Nested lists (ordered)" examples.

```
{
    ol: [
        'Item 1',
        'Item 2 - Lorem ipsum dolor sit amet, consectetur adipisicing elit. Malit profecta versatur nomine ocurreret multavit',
        [
            'Item 3',
            {
                ul: [
                    'subitem 1',
                    'subitem 2',
                ]
            },
        ],
        'item 4\nsecond line of item3',
    ]
},
```

**Results in**
1. Item 1
2. Item 2 - Lorem ipsum dolor sit amet, consectetur adipisicing elit. Malit profecta versatur
nomine ocurreret multavit
3. Item 3
   * subitem 1
   * subitem 2
4. item 4
second line of item3

**Comment by liborm85** [comment]
Fixed by PR https://github.com/bpampuch/pdfmake/pull/2760.
Released in version 0.2.11.
